ecommerce/templates/edx/checkout/receipt.html

```diff
@@ -106,6 +106,11 @@ <h2>{% trans "Order Information" as tmsg %}{{ tmsg | force_escape }}</h2>
                 <dd class="line-price price">{{ line.unit_price_incl_tax|currency:order.currency }}</dd>
               </div>
             {% endfor %}
+             <form style="display: none;"  name="aw_basket_form">
```

Comment:
Use class `hidden` instead of inline styles.

Reply:
Also, move this under if clause:
```suggestion
        {% if product_tracking %}
             <form style="display: none;"  name="aw_basket_form">
```

Reply:
Class hidden does not work.

Reply:
It does work, I have tested locally `class="hidden"`.